### PR TITLE
feat(learn): enable ukrainian certifications

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

**Comment by raisedadead** [comment]
Hey @nhcarrigan - can you confirm these are the only changes to "code", that is required?

**Comment by raisedadead** [comment]
Never mind, the tests speak what's needed. Whoever wrote those is a rockstar.

**Comment by raisedadead** [comment]
The next steps are to coordinate with the team to get the missing files in place. cc @freeCodeCamp/i18n

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
We need to add Ukrainian to the `/client/src/utils/algolia-locale-setup.ts` file.

**Review by naomi-lgbt — DISMISSED** [review]
With the latest commit this builds locally and appears to function as intended. LGTM.

**Comment by raisedadead** [comment]
Great so I am writing a flight manual for this, and these are the only changes then, to enable a language. That is these changes can assume all other files curriculum, meta data like quotes, etc. are in place.

**Comment by nhcarrigan** [comment]
FYI we already have a flight manual: https://contribute.freecodecamp.org/#/how-to-test-translations-locally?id=enabling-a-language

**Comment by nhcarrigan** [comment]
A couple of the steps in that flight manual were a bit outdated - I've [got a PR](https://github.com/freeCodeCamp/freeCodeCamp/pull/44464) to update that.

**Comment by raisedadead** [comment]
Ah - I forgot about it. Thanks.

File: config/i18n/all-langs.ts [hunk 1]
```diff
@@ -92,6 +94,20 @@ export const auditedCerts = {
...
+  ukrainian: [
+    SuperBlocks.RespWebDesign,
+    SuperBlocks.JsAlgoDataStruct,
+    SuperBlocks.FrontEndDevLibs,
+    SuperBlocks.DataVis,
+    SuperBlocks.BackEndDevApis,
+    SuperBlocks.QualityAssurance,
+    SuperBlocks.SciCompPy,
+    // SuperBlocks.DataAnalysisPy,        // These are not yet approved atleast 95%
+    // SuperBlocks.InfoSec,               // These are not yet approved atleast 95%
```
Comment (ieahleen) [inline config/i18n/all-langs.ts:107]: I think they can go in anyway, it's the importing of the project description to the curriculum that caused this reduction. It happened in all languages, and if the Ukrainian team didn't notice I think they can just been told and will be fast

Or the following certs need to wait too, as we want to have the certs published in order

Comment (raisedadead) [inline config/i18n/all-langs.ts:107]:
```suggestion
    SuperBlocks.DataAnalysisPy,
    SuperBlocks.InfoSec,
```

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**